ons_alpha/bulletins/models.py

```diff
@@ -61,22 +57,25 @@ class BulletinPage(BasePage):
         ]
     )
 
-    def get_context(self, request, *args, **kwargs):
-        context = super().get_context(request, *args, **kwargs)
-
-        context["toc"] = self.toc
-
-        return context
+    @property
+    def is_latest(self):
+        latest_id = (
+            BulletinPage.objects.sibling_of(self).live().order_by("-release_date").values_list("id", flat=True).first()
+        )
+        return self.pk == latest_id
 
-    @cached_property
-    def toc(self):
-        items = [{"url": "#summary", "text": "Summary"}]
 
-        for block in self.body:  # pylint: disable=not-an-iterable
-            if hasattr(block.block, "to_table_of_contents_items"):
-                items += block.block.to_table_of_contents_items(block.value)
+class BulletinSeriesPage(RoutablePageMixin, Page):
+    template = "templates/pages/bulletins/series_page.html"
+    parent_page_types = ["home.HomePage"]
+    subpage_types = ["BulletinPage"]
 
-        if self.contact_details_id:
-            items += [{"url": "#contact-details", "text": "Contact details"}]
+    @route(r"^latest/$")
```

Comment:
best to use `wagtail.contrib.routable_page.models.path`

```suggestion
    @route("latest/")
```

as you don't need any regex-ing.

The import will then have to change from `from wagtail.contrib.routable_page.models import RoutablePageMixin, route` to `from wagtail.contrib.routable_page.models import RoutablePageMixin, path`